no-computer.md

```diff
@@ -58,12 +104,32 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+_Once someone got an idea to work on this project, they would have to decide which
+pieces they want to include in their DSL. So, this would all be engaging with the
+language because they would have to think about how the blocks can interact with
+each other, how the colors interact with each other, and how many of each block is
+enough to make interesting patterns without being overwhelming. The "systems" aspect
+of this project would just be making the blocks, which would mostly require some
+hours in the machine shop (or an engineering friend)._
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
-
+_This project does not have a huge scope because the project as shown is about 5
+shapes of blocks, each of which has its own color. If someone wanted to add
+to this project, they could make it bigger. However, they would have to carefully
+consider which shapes to add._
```

Comment:
Color choice would also be important. Other than having a color scheme that is aesthetically pleasing, it would be good design to make the set of colors distinguishable for all types of color-blindness.